lib/generators/statesman/active_record_transition_generator.rb

```diff
@@ -27,5 +27,9 @@ def migration_file_name
     def rails_4_or_higher?
       Rails.version.split(".").map(&:to_i).first >= 4
     end
+
+    def rails_5_or_higher?
```

Comment:
You shouldn't need this function once it's in `GeneratorHelpers`. Would be nice to move `rails_4_or_higher?` into there, too.